.github/workflows/pr_test_cpu.yml

```diff
@@ -12,12 +12,12 @@ concurrency:
 jobs:
   tests-cpu:
     name: ${{ matrix.os }} - python-${{ matrix.python-version }}, torch-${{ matrix.pytorch-version }}, ${{ matrix.pytorch-dtype }}
-    runs-on: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}-latest
 
     strategy:
       fail-fast: true
       matrix:
-        os: [ubuntu-latest, windows-latest, macos]
```

Comment:
form another package i maintain: `os: [ubuntu-latest, windows-latest, macos-latest]`
https://github.com/farm-ng/farm-ng-core/blob/main/.github/workflows/ci_python.yml#L17

Reply:
What I did here was just to have the OS names and add the version ("latest") at `run-on:` -- this decreases the name of the job and the codecov flags